projects/igniteui-angular/src/lib/grids/grid/grid.search.spec.ts

```diff
@@ -720,6 +720,34 @@ describe('IgxGrid - search API', () => {
 
             expect(cell.nativeElement.innerText.trim()).toBe('Casey Houston');
         });
+
+        it('Search should not change the cell\'s value', async () => {
+            grid.findNext('12');
+            const rowIndexes = [1, 3, 4, 5];
+
+            rowIndexes.forEach(ind => {
+                const cell = grid.getCellByColumn(ind, 'HireDate');
+                const highlights = cell.nativeElement.querySelectorAll('.' + fix.componentInstance.highlightClass);
+                const activeHighlight = cell.nativeElement.querySelector('.' + fix.componentInstance.activeClass);
+                const cellChildren = cell.nativeElement.children;
+
+                // Check whether search does not change the cell's value
+                expect(cellChildren.length).toBe(2);
+                expect(cellChildren[0].innerText).toEqual(cell.value);
```

Comment:
Checking the cell's children innertext is OK, but we are more interested in the cell's innerText in order to be sure it's not doubled.